### PR TITLE
Wikia has re-branded as FANDOM

### PR DESCRIPTION
**Pull request by ghost**
Wikia re-branded as FANDOM quite a while ago, so it's definitely worth updating this.

### CODE REVIEW

**Comment by kmpoppe** [comment]
Thanks for the PR.

The logo looks like it could benefit from being put through tinypng.com - would you please compress the image and replace that in your branch?

**Comment by phallobst** [comment]
#4290 has been lingering for quite some time now... There seems to be no agreement on whether to move Wikia/Fandom to "Social" or not. Also, #4837 has already been closed as being a duplicate. Any opinions by the @2factorauth/collaborators?

**Comment by kmpoppe** [comment]
You're right Michael, I thought there was a déjà vu...

I'm fine with moving it to Social.

K

Michael Loßin <notifications@github.com> schrieb am Do., 6. Aug. 2020,
09:24:

**Comment by ghost** [comment]
> Thanks for the PR.
> 
> The logo looks like it could benefit from being put through tinypng.com - would you please compress the image and replace that in your branch?

Sure thing

File: _data/other.yml [hunk 1]
```diff
@@ -174,6 +174,11 @@ websites:
     img: extremenetworks.png
     twitter: ExtremeNetworks
     facebook: extremenetworks
+    
+  - name: FANDOM
+    url: https://www.fandom.com/
+    twitter: wikia
```
Comment (Carlgo11) [inline _data/other.yml]:
```suggestion
    twitter: getFANDOM
    facebook: getFANDOM
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Review by jamiehowarth0 — APPROVED** [review]

**Review by jamcat22 — APPROVED** [review]

**Comment by jamcat22** [comment]
Merging this to address rebranding part. If we move Wikipedia in #4286, then we can also move FANDOM as well.

**Pull request merged**